app/api/students/info.py

```
@@ -0,0 +1,27 @@
+class StudentInformationController:
+    usecase: StudentInformationUseCase = Depends(get_student_information_usecase)
+
+    @router.post(
+        path="/info",
+        summary="학생 정보",
+        description="학교 시스템에 로그인하여 학생 정보를 불러옵니다.",
+        response_class=ORJSONResponse,
+        response_model=StudentInformationResponse,
+        status_code=status.HTTP_200_OK,
+    )
+    async def get_student_information(self) -> StudentInformationResponse:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)